app/javascript/components/artists/UpdateArtist.jsx

```diff
@@ -158,20 +175,42 @@ class UpdateArtist extends React.Component {
             required
           />
           <FormError error={this.state.errors.name}/>
-          <h5>Program</h5>
+          <h5>Degree</h5>
           <select
-            value={this.state.artist.program}
+            value={this.state.artist.degree ? this.state.artist.degree : 0}
             onChange={this.handleChange}
-            name="program"
-            className="input-dropdown ttc"
+            name="degree"
+            className="input-dropdown ttu"
             required
           >
             {
-              Object.keys(this.state.categories.program).map((obj, i) => {
+              Object.keys(this.state.categories.degree).map((obj, i) => {
                 return <option key={i} value={obj}>{convertSnakeCase(obj)}</option>
               })
             }
           </select>
+          <h5>Program</h5>
+
+          <div className="checkbox-container">
+                  {Object.keys(this.state.categories.program).map(item => (
+                    <div className="mb2 checkbox-item" key={item}>
+                      <label className="ttc dib flex" htmlFor={`checkbox-${item}`}>
+                        <input
+                          onChange={() => this.toggleCheckbox(item)}
+                          type="checkbox"
+                          className="checkbox"
+                          value={convertSnakeCase(item)}
+                          id={`checkbox-${item}`}
+                          name="program"
+                          checked={(this.state.artist['program'].includes(item)) ? true : false}
+                        />
+                        <span className="filter-item">
```

Comment:
We can remove `filter-item`, it just constraints the width here which isn't necessary for this page